File: acm/ACM/cf/TheatreSquare.py
n,m,a = map(int,input().split(" "))

sum=0
sum+=int(n/a)
if n%a!=0:
    sum+=1
sum1=0
sum1+=int(m/a)

if m%a!=0:
    sum1+=1
print(int(sum*sum1))

File: acm/ACM/cf/TheatreSquare.cpp
#include <iostream>

using namespace std;

int main()
{
    long long int n, m, a;
    while (cin >> n >> m >> a)
    {
        long long int sum = 0;
        sum += n / a;
        if (n % a != 0)
            sum++;
        long long int sum1 = 0;
        sum1 += m / a;
        if (m % a != 0)
            sum1++;
        cout << sum * sum1 << endl;
    }
}